get_aqi.py

```diff
@@ -1,32 +1,55 @@
 #!/usr/bin/python3
 
+import os
 import argparse
 import datetime
 import time
+import logging
```

Comment:
I prefer to sort imports :smile: 

```suggestion
import argparse
import datetime
import logging
import os
import time
```